napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -209,6 +210,11 @@ def record_annotations(self, record_idx: int):
             self._annotation_model.add_annotation(
                 self._annotation_model.get_all_images()[record_idx], self.view.get_curr_annots()
             )
+            # self._annotation_model.clear_all_cur_img_points_layers()
+        if record_idx == self._annotation_model.get_previous_image_index():
+            self._annotation_model.annotation_saved()
```

Comment:
leave a comment

Reply:
comment in https://github.com/aics-int/napari-allencell-annotator/pull/94/commits/5118bdd12621d4b00ddea357ea03c32f00f7eb72